kitsune/sumo/static/js/gallery.js

```diff
@@ -203,11 +208,15 @@
             var file = $input[0].files[0],
                 type = $input.attr('name');
             var file_ext = file.name.split(/\./).pop().toLowerCase();
-            return (in_array(file_ext, CONSTANTS.extensions[type]) &&
-                    file.size < CONSTANTS.max_size[type]);
+            return (in_array(file_ext, CONSTANTS.extensions[type]));
+        },
+        isNotTooLarge: function ($input) {
```

Comment:
It is odd that this is `isNotTooLarge`, and you only ever call it like `!isNotTooLarge(file)`. It is like a double negative. It would make more sense to have it be `isTooLarge()`, and call it directly. I'm also not convinced it needs to be a separate function, since it is only a couple lines and isn't re-used anywhere.